surfsense_backend/app/config/uvicorn.py

```
@@ -0,0 +1,82 @@
+        "UVICORN_FD": ("fd", lambda x: _parse_int(x, "UVICORN_FD")),
+        "UVICORN_ROOT_PATH": ("root_path", str),
+    }
+
+    # Process advanced configuration options
+    for env_var, (config_key, parser) in config_mapping.items():
+        value = os.getenv(env_var)
+        if value:
+            try:
+                config_kwargs[config_key] = parser(value)
+            except ValueError as e:
+                raise ValueError(f"Configuration error for {env_var}: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)